tests/test_jira.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from unittest.mock import patch
+from goose.toolkit.jira import Jira
+
+
+class TestJiraToolkit(unittest.TestCase):
+    @patch("goose.toolkit.jira.Message.load")
+    def test_jira_system_prompt(self, mock_load):
```

Comment:
this mocking doesn't skips the formatting piece, so this doesn't check it i believe. We can instantiate the class directly here and check instance.system()

Reply:
ah oops...